src/components/HSButton/HSButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import '../../styles/index.scss';
+
+import { ButtonVariant, HSButtonProps } from './types';
+
+const HSButton: React.FC<HSButtonProps> = ({
+  label = 'Click Me!',
+  icon,
+  type = null,
+  color = null,
+  link,
+  onClick,
+  children
+}) => {
+  const ButtonBase: React.ReactElement = (
+    <>
+      {icon && <i className='hs-button__icon'>{icon}</i>}
+      {label || (children && <span className='hs-button__label'>{label}</span>)}
```

Comment:
Can we have the default value of the button label to be the children? I think that makes it for a better DX. (or am I just reading the code wrong lmao)

Reply:
I have removed label prop and gave children type - string | ReactChild 
I hope this is what you wanted 😄